lib/spaceship/secure_net_http.rb

```diff
@@ -0,0 +1,55 @@
+require 'net/http'
+
+# https://bugs.ruby-lang.org/issues/9450
+# monkey patched...
+unless Net::HTTP::SSL_IVNAMES.include? @ssl_options
+  puts "Monkey patch Net:HTTP to support SSL options setting"
+  module Net
+    class HTTP
+      Net::HTTP::SSL_IVNAMES << :@ssl_options
+      Net::HTTP::SSL_ATTRIBUTES << :options
+
+      # Sets the SSL options flags. See OpenSSL::SSL::SSLContext#options=
+      attr_accessor :ssl_options
+    end
+  end
+end
+
+require 'faraday'
+module Faraday
+  class Adapter
+  	# Harden default NetHttp for SSL
+	# http://www.safe-t.com/ruby-and-ssl/
+    class SecureNetHttp < NetHttp
+      def configure_ssl(http, ssl)
+        super
+
+        puts "Hardening NetHttp into SecureNetHttp"
+        #puts Net::HTTP::SSL_IVNAMES
+        #puts Net::HTTP::SSL_ATTRIBUTES
+
+        #http.ciphers = ssl[:ciphers]              if ssl[:ciphers]
+        #http.ssl_options = ssl[:ssl_options]      if ssl[:ssl_options]
```

Comment:
Incorrect indentation detected (column 8 instead of 2).<br>Missing space after #.